dashboard/src/components/Footer.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <footer>
+    <p>
+      Coded with lots of <img src="https://discord.com/assets/25c09e6fde32411da2b0da00f5cb9c84.svg" alt="coffee_logo"> by 
+      <a href="https://npstr.space/">Napster</a>
+    </p>
+  </footer>
+</template>
+
+<style lang="scss">
+  footer {
+
+    font-size: 13px;
```

Comment:
Bulma comes with a bunch of css classes for this. See https://github.com/wolfiabot/Wolfia/blob/master/dashboard/src/components/LogIn.vue#L20 for example